openquakeplatform/openquakeplatform/static/js/irv_viewer_d3_tree.js

```diff
@@ -221,32 +275,25 @@
                 .attr("class", (function(d) { return "level-" + d.level; }))
                 .attr("id", "svg-text")
                 .attr("value", (function(d) { return d.weight; }))
-                .attr("x", function(d) { return -(d.weight * CIRCLE_SCALE + 5); })
+                .attr("x", function(d) { return -(getRadius(d) + 5); })
                 .attr("dy", function(d) {
                     // NOTE are x and y swapped?
                     // set te text above or below the node depending on the
                     // parent position
-                    if (typeof d.parent != "undefined" && d.x > d.parent.x){
+                    if (typeof d.parent != 'undefined' && d.x > d.parent.x){
                         return "2em";
                     }
                     return "-1em";
                 })
                 .attr("text-anchor", function(d) { return "end"; })
                 .text(function(d) {
-                    return d.name;
+                    if (d.weight < 0) {
```

Comment:
We don't check with respect to the weight's sign anymore, because now we can use d.isInverted